File: ruby/elements/html.rb
module Elements
  class Html < Element
    def initialize( compiler, article, lines)
      @lines = lines
    end

    def line_count
      100
    end

    def to_data( compiler, article)
      {'type' => 'raw', 'text' => @lines.join( "\n")}
    end
  end
end

File: java/src/com/alofmethbin/jadawin/elements/Html.java
package com.alofmethbin.jadawin.elements;

import com.alofmethbin.jadawin.Article;
import java.util.List;

public class Html extends Element {
    private List<String> lines;
    
    public Html( Article article, List<String> lines) {
        super( article);
        this.lines = lines;
    }

    @Override
    public int lineCount() {
        return 100;
    }
}
